uninstall.sh

```diff
@@ -42,14 +42,19 @@ opt_skip_cache_and_logs=""
 failed=false
 
 un=$(uname)
+unm=$(uname -m)
 case "$un" in
   Linux)
     ostype=linux
     homebrew_prefix_default=/home/linuxbrew/.linuxbrew
   ;;
   Darwin)
     ostype=macos
-    homebrew_prefix_default=/usr/local
+    if [[ "$unm" == "arm64" ]]; then
```

Comment:
```suggestion
    if [[ "$(uname -m)" == "arm64" ]]; then
```

Reply:
Added New commit to the pr resolving above suggestions. Thank you.